lib/travis/api/v3/remote_query.rb

```diff
@@ -56,7 +56,16 @@ def config
     end
 
     def s3_config
-      raise NotImplemented
+      conf = config.log_options.try(:s3) || {}
```

Comment:
This gets the log_options from config. However, we somehow need to have the remote query know whether to look for log_options or cache_options. Not sure how to do this...